vars/signArtifacts.groovy

```diff
@@ -14,6 +17,8 @@ void call(Map args = [:]) {
 
     importPGPKey()
 
+    String arguments = generateArguments(args)
```

Comment:
```suggestion
    
// Since the first argument should be artifactPath
String arguments = args.artifactPath

args.each(entry -> entry.key.equals('artifactPath') ? "" : arguments += " --${entry.key}=${entry.value}")

```

Reply:
Ok, this seems to be more concise.